src/bh/components/BhText.vue

```diff
@@ -67,7 +67,7 @@ export default {
   },
   mounted() {
     const { input } = this.$refs;
-
+    this.formatters.reduce((c, f) => f(c), this.value || '');
```

Comment:
This line do nothing...

Maybe do something:
```
this.val = this.formatters.reduce((c, f) => f(c), this.value || '');
```

Reply:
i just fix it haha